integration-tests/src/test/java/oracle/kubernetes/operator/utils/Secret.java

```diff
@@ -58,6 +59,55 @@ public Secret(String namespace, String secretName, String username, String passw
     LoggerHelper.getLocal().log(Level.INFO, "command result " + result.stdout().trim());
   }
 
+  /**
+   * Construct secret.
+   * @param namespace namespace
```

Comment:
How about something like " @param namespace namespace where the secret is going to create"

Reply:
good idea. Will fix

Reply:
fixed